ui/lib/pki/addon/components/page/pki-role-details.hbs

```diff
@@ -72,7 +72,7 @@
             @value={{val}}
             @alwaysRender={{true}}
           />
-        {{else if (eq attr.name "noStore")}}
+        {{else if (includes attr.name (array "noStore" "noStoreMetadata"))}}
```

Comment:
Because the attribute is only added to the model for enterprise versions - we don't have to worry about this field rendering for community versions, right?

Reply:
Correct! The filtering is happening on the model level